adcm_client/base.py

```diff
@@ -69,6 +73,28 @@ class PagingEnds(Exception):
     """There are no more data in paginated mode."""
 
 
+class IncompatibleVersion(Exception):
+    """Incompatible version, upgrade version ADCM."""
+    def __init__(self, method_name='', version='', message=None):
+        if message is None:
+            self.message = (f'The "{method_name}" method works with versions older {version},'
+                            f' upgrade version ADCM.')
+        else:
+            self.message = message
+        super(IncompatibleVersion, self).__init__(self.message)
+
+
+def version_compatibility_check(version):
+    def decorate(func):
+        @wraps(func)
+        def wrapper(*args, **kwargs):
+            if rpm.compare_versions(args[0].api_version, version) < 0:
+                raise IncompatibleVersion(func.__name__, version)
```

Comment:
Same things about IcompatibleVersion.  Word "Icompatible" too wide. If some day you will write max version check. You will have two exception: "Max" for maximum and "Incompatible" for minimun. It is better to call this like "TooOldServer" or something like this.

Reply:
Ok